permission-portal-frontend/client/src/store/index.js

```diff
@@ -43,6 +43,8 @@ const createStore = (WrappedComponent) => {
                         this.organizationMembersListener = db
                           .collection('users')
                           .where('organizationID', '==', rootStore.organization.id)
+                          // .orderBy('lastName')
+                          // .orderBy('firstName')
```

Comment:
Commented out because we have to [create an index][index] first.

[index]: https://firebase.google.com/docs/firestore/query-data/indexing